test/comment-directives.js

```diff
@@ -38,7 +51,7 @@ describe('Linter', () => {
       const report = linter.processStr(
         `
                 /* solhint-disable-next-line */
-                pragma solidity ^0.4.4; 
+                pragma solidity ^0.4.4;
```

Comment:
Looks like these had some empty whitespace at the end that my editor removed automatically. I can roll these changes back if that's an issue.

Reply:
No, I'm all for removing trailing whitespace :smile:

Reply:
No, that is absolutely fine. Thanks.